app/js/components/orderEntry/addForm/DraftDataTable.jsx

```diff
@@ -184,7 +184,7 @@ export class DraftDataTable extends React.Component {
             <i className="icon-edit" title="Edit" />
           </a>
           <a id="discard-draft-order" href="#" onClick={() => handleDiscardOneOrder(order)}>
-            <i className="icon-remove" title="Delete" />
+            <i className="icon-remove" title="Discontinue" />
```

Comment:
You should use discard for this part

Reply:
Would have to consult the team to get more feedback on this.